bare-metal/apple-silicon/how-to/use-private-networks.mdx

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  title: How to use Private Networks with your Mac mini
+  description: Learn how to get the most from the Private Networks 1Gbps feature for your Mac mini. Find out how to activate the feature, attach your Mac mini to Private Networks, and enable secure communication with your other Scaleway resources.
+content:
+  h1: How to use Private Networks with your Mac mini
+  paragraph: Learn how to get the most from the Private Networks 1Gbps feature for your Mac mini. Find out how to activate the feature, attach your Mac mini to Private Networks, and enable secure communication with your other Scaleway resources.
+tags: mac-mini mac-mini apple-silicon private-network vpc
+dates:
+  validation: 2024-12-19
+  posted: 2021-05-26
+categories:
+  - bare-metal
+---
+
+Private Networks allow your Mac mini to communicate in an isolated and secure network with other Scaleway resources, away from the public internet. This is an optional-billed feature for Mac mini.
+
+Each Mac mini can be connected to one or several Private Networks, letting you build your own network topologies. Private Networks’ built-in DHCP server manages the allocation of private IP addresses for Mac mini servers (and other resources) when they are attached to the network.
+
+For full information about Scaleway Private Networks and VPC, see our [dedicated documentation](/network/vpc/)
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- A [Mac mini](/bare-metal/apple-silicon/how-to/create-mac-mini/)
+
+## How to enable the Private Networks feature
+
+Before you can attach your Mac mini to a Private Network, you must enable the feature. This can be done during [Mac mini creation](/bare-metal/apple-silicon/how-to/create-mac-mini/) or afterwards. These instructions show how to enable the feature after creation of the Mac mini.
```

Comment:
```suggestion
Before you can attach your Mac mini to a Private Network, you must enable the feature. This can be done during [Mac mini creation](/bare-metal/apple-silicon/how-to/create-mac-mini/) or afterwards. These instructions show how to enable the feature after the creation of the Mac mini.
```